### PR TITLE
Add light sensor to peripherals

### LINKED ISSUE #40: Light sensor not included in library
The light sensor is not included in the MagTag library. It should be added as it is a feature on the board.

### PR DESCRIPTION
**Pull request by makermelissa**
Fixes #40.

### CODE REVIEW

File: adafruit_magtag/peripherals.py [hunk 1]
```diff
@@ -143,3 +146,11 @@ def any_button_pressed(self):
         Return whether any button is pressed
         """
         return False in [self.buttons[i].value for i in range(0, 4)]
+
+    @property
+    def light(self):
+        """
+        Return the value of the light sensor. The neopixel_disable property
+        must be false to get a value.
+        """
```
Comment (kattni) [inline adafruit_magtag/peripherals.py:155]: Consider adding the example to this  docstring.
```suggestion
        .. code-block:: python
            import time
            from adafruit_magtag.magtag import MagTag

            magtag = MagTag()

            while True:
                print(magtag.peripherals.light)
                time.sleep(0.01)
        """
```

**Review by kattni — CHANGES_REQUESTED** [review]
Looks great! Thanks for adding this. Please see the suggestion below.

**Comment by makermelissa** [comment]
Good suggestion. Added.

**Review by kattni — APPROVED** [review]

**Comment by kattni** [comment]
@makermelissa Thanks for updating this! Feel free to merge once checks pass.

**Pull request merged**